<ui version="4.0" >
 <author>Frank Büttner</author>
 <class>dlgEinstellungenBasis</class>
 <widget class="QDialog" name="dlgEinstellungenBasis" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Einstellungen</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListWidget" name="lstGruppe" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentRow" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>Verzeichnisse</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Manifeste</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Mergemodule</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Zertifikat</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="staGruppe" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>45</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="Verzeichnisse" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Verzeichnisse</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignHCenter</set>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>Pfad zum "lib" Verzeichnis von Qt:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtQtPfad" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>C:\c++\Qt\4.2.2MS\lib</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sfQtPfadSuchen" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="baseSize" >
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Pfad zum Wix Verzeichnis:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtWixPfad" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>C:\c++\WiX3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sfWixPfadSuchen" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_8" >
                <property name="text" >
                 <string>Pfad zum Windows SDK:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtWindowsSDKPfad" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip" >
                 <string>Hier bitte den Pfad eingeben, wo sich die Werkzeuge aus dem Windows SDK befinden.</string>
                </property>
                <property name="text" >
                 <string>C:\Programme\Microsoft SDKs\Windows\v6.0\Bin</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sfWindowsSDKPfadSuchen" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4" >
                <property name="text" >
                 <string>Zielverzeichnis für die Mergemodule:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtZielPfad" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>f:\temp\Qt4\Mergemodule</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sfZielpfadSuchen" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Manifeste" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Manifest</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignHCenter</set>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5" >
                <property name="text" >
                 <string>Name des Entwicklers/Firma:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtEntwicklername" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>FrankBuettner</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6" >
                <property name="text" >
                 <string>Zielplatform:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="awProzessor" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text" >
                  <string>x86</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>x64</string>
                 </property>
                </item>
                <item>
                 <property name="text" >
                  <string>ia64</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Mergemodule" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>Mergemodule</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_9" >
                <property name="text" >
                 <string>Autor:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMergemodulAutor" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Frank Büttner</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Zertifikate" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Zertifikat</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignHCenter</set>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>Zu benutzendes Zertifikat:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtZertifikat" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Dies ist der SHA-1 Fingerabdruck des Zertifikates.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">Damit wird das Zertifikat später identifiziert.&lt;/p>&lt;/body>&lt;/html></string>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sfZertifikatSuchen" >
                <property name="text" >
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>PublickeyToken:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtPublickeyToken" >
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>255</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip" >
                 <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Der publickeyToken läßt sich aus dem Zertifikat z.B. mit dem Programm &lt;/span>Pktextract aus dem Platform SDK ermitteln.&lt;/p>&lt;/body>&lt;/html></string>
                </property>
                <property name="text" >
                 <string>c0677197e04ed00a</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="sfBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lstGruppe</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>staGruppe</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>103</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel" >
     <x>379</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
